underscore.js

```diff
@@ -441,6 +444,7 @@
   // Return the number of elements in an object.
   _.size = function(obj) {
     if (obj == null) return 0;
+    if (supportToSetAndMap && (obj instanceof Set || obj instanceof Map)) return obj.size;
```

Comment:
This is still using `instanceof` which will fail sets and maps from other realms (think iframe / `vm` module in Node).